docs/plan/componentize-golems/phase-4-bun-workspaces/README.md

```
@@ -0,0 +1,48 @@
+# Phase 4: Bun Workspaces
+
+> [Back to main plan](../README.md)
+
+## Goal
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)